src/Features/Core/Portable/xlf/FeaturesResources.cs.xlf

```diff
@@ -67,6 +67,11 @@
         <target state="new">Avoid unused paramereters in your code. If the parameter cannot be removed, then change its name so it starts with an underscore and is optionally followed by an integer, such as '_', '_1', '_2', etc. These are treated as special discard symbol names.</target>
         <note />
       </trans-unit>
+      <trans-unit id="CantApplyChangesUnexpectedError">
+        <source>Can't apply changes -- unexpected error: '{0}'</source>
```

Comment:
>Can't [](start = 16, length = 5)

We use 'cannot' more often than 'can't'. It seems to be preferable.

Reply:
Fixed.